EDDiscovery/EDDiscoveryForm.cs

```diff
@@ -936,7 +936,10 @@ public void StoreSystemNote(HistoryEntry he, string txt, bool send = false)
                 // Also notify various "quick" listeners
                 if (he.UpdateSystemNote(txt, send))
                 {
-                    _uncommittedNoteHistoryEntry = he;
+                    if (_uncommittedNoteHistoryEntry == null)
```

Comment:
Only need to set it once since it's a reference and it will be automatically sent in the next section if send==true. I wanted to avoid duplicating code since we still need the check for an uncommitted note (although not 100% sure whether that can actually happen now).


Overall I think the whole note-saving probably needs a bit of a refactor, especially if we implement the per-jump notes etc. For now I wanted to make the least possible changes to avoid introducing new bugs (and failing...)